dashboard/test/ui/features/step_definitions/pd.rb

```diff
@@ -114,22 +167,22 @@
 And(/^I make the teacher named "([^"]*)" a facilitator for course "([^"]*)"$/) do |name, course|
   require_rails_env
 
-  user = User.find_by(name: name)
+  user = User.find_by(email: @users[name][:email])
```

Comment:
name is not unique, but email is, so this formerly had the potential to find the wrong user